src/adloader.js

```diff
@@ -50,6 +54,11 @@ const _approvedLoadExternalJSList = [
  * @param {object} attributes an object of attributes to be added to the script with setAttribute by [key] and [value]; Only the attributes passed in the first request of a url will be added.
  */
 export function loadExternalScript(url, moduleCode, callback, doc, attributes) {
+  if (!isActivityAllowed(LOAD_EXTERNAL_SCRIPT, activityParams(MODULE_TYPE_PREBID, 'adLoader'))) {
```

Comment:
I think to be useful the module should be the one attempting to load the script (rather than prebid/adloader). Otherwise I can only disable all scripts or none - the activity parameters are always the same. It does require some refactoring to get the caller's module type.